packages/editor/src/components/parts/common/info/Information.tsx

```diff
@@ -46,6 +47,15 @@ const Information = <T extends InformationConfig>({ config, update }: Informatio
     })
   ];
 
+  const getCMSUrl = (category: string) => {
+    let middlePart = category;
+    if (category.includes('<%=')) {
```

Comment:
was this supported in the old swt inscription? open a macro cms path?

Reply:
yes, in the old SWT Inscription, clicking the button while a macro was selected would add it to the CMS under Categories/MACRO/name.

Reply:
Sounds like a bug to me...
Or did bruno confirm this was a feature and was used?

Reply:
Yes, thats possible. I haven't asked him yet, as he was still on vacation last week, but I can definitely check with him.